app/views/devices/index.html.erb

```diff
@@ -0,0 +1,84 @@
+          <%= image_tag("illustrations/ill-keurig.svg", :alt => "Keurig") %>
+          <h3>Keurig</h3>
+        </a>
+      </li>
+
+      <li class="device__coffee-maker">
+        <a href="/device-coffee-maker">
+          <%= image_tag("illustrations/ill-coffee-maker.svg", :alt => "Coffee Maker") %>
+          <h3>Coffee Maker</h3>
+        </a>
+      </li>
+
+      <li class="device__french-roast">
+        <a href="/device-french-roast">
+          <%= image_tag("illustrations/ill-french-roast.svg", :alt => "French Roast") %>
+          <h3>French Roast</h3>
+        </a>
+      </li>
+    </ul>
+  </nav>
+
+  <section class="device__problems">
+    <ul>
+      <li>
+        <a href="">
+          <h4>Lukewarm coffee chills my soul</h4>
+          <%= image_tag("icon-caret-hover.svg") %>
+        </a>
+        <div class="problem__solutions">
+          <p><span>Hack: do a pre-cycle.</span>Running a water cycle before brewing will give your machine a chance to warm up. In addition, you should avoid paper or foam cups at all costs — they lose heat about as quickly as you lose the ability to try.</p>
+        </div>
+      </li>
+
+      <li>
+        <a href="">
+          <h4>Lukewarm coffee chills my soul</h4>
+          <%= image_tag("icon-caret-hover.svg") %>
+        </a>
+        <div class="problem__solutions">
+          <p><span>Hack: do a lazy version of descaling.</span>Pour a 1:2 ratio of vinegar to water in your Keurig’s reservoir and run it until it’s empty.  Repeat every three to six months (translation: every year).</p>
+        </div>
+      </li>
+
+      <li>
+        <a href="">
+          <h4>Lukewarm coffee chills my soul</h4>
+          <%= image_tag("icon-caret-hover.svg") %>
+        </a>
+        <div class="problem__solutions">
+          <p><span>Hack: use the same k-cup twice</span> Yep, you heard us: brew two cups with the same k-cup. The watery second brew will balance out the strength of the first cup.</p>
+        </div>
+      </li>
+
+      <li>
+        <a href="">
+          <h4>Lukewarm coffee chills my soul</h4>
+          <%= image_tag("icon-caret-hover.svg") %>
+        </a>
+        <div class="problem__solutions">
+          <p><span>Hack: do a pre-cycle.</span> Running a water cycle before brewing will give your machine a chance to warm up. In addition, you should avoid paper or foam cups at all costs — they lose heat about as quickly as you lose the ability to try.</p>
+        </div>
+      </li>
+
+      <li>
+        <a href="">
+          <h4>Lukewarm coffee chills my soul</h4>
+          <%= image_tag("icon-caret-hover.svg") %>
+        </a>
+        <div class="problem__solutions">
+          <p><span>Hack: do a pre-cycle.</span> Running a water cycle before brewing will give your machine a chance to warm up. In addition, you should avoid paper or foam cups at all costs — they lose heat about as quickly as you lose the ability to try.</p>
+        </div>
+      </li>
```

Comment:
This is a lot different from what you have, I know. I can walk you through why I did it this way.